Gemfile.lock

```diff
@@ -67,6 +67,7 @@ GEM
     zeitwerk (2.6.7)
 
 PLATFORMS
+  arm64-darwin-21
```

Comment:
Thoughts on adding m1?

Reply:
yeah it's ok. Its just a lockfile so for missing platforms it will just materialize itself if missing.